ldes-fragmentisers/ldes-fragmentisers-common/src/test/resources/features/fragmentdeletion.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Execute FragmentDeletion
+
+  Scenario: Execution FragmentDeletion
+    Given the following Fragments are present
+      | fragmentIdentifier                        | immutable | nrOfMembersAdded | relation                                               | nrOfDaysInFuture |
```

Comment:
Maybe this could be more explicit if "nrOfDaysInFuture" was something like "daysUntilDeletion" ? At first I had no idea what this variable was.

Reply:
I also would just reject null values for this because this is more logic that people need to figure out without any added value.